src/presentation/index.ts

```diff
@@ -63,6 +63,7 @@ function startRundownEventServer(): void {
 function startSystemServices(): void {
   ServiceFacade.createIngestChangeService()
   ServiceFacade.createIngestService()
+  ServiceFacade.createMongoDatabaseChangeService()
```

Comment:
Piggybacking on the comment from the business logic layer facade.
This makes it so that we have a specific database reference (which should be an implementation detail of the data access layer) in the index.ts file...

Reply:
A good point, it has been changed in reference to your earlier comment on it!